sys/include/net/lwm2m.h

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (C) 2018 Beduino Master Projekt - University of Bremen
+ *               2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup     net_lwm2m LWM2M
+ * @ingroup      net
+ * @brief        Lightweight Machine to Machnine (LWM2M) definitions
+ * @{
+ *
+ * @file
+ * @brief       LWM2M header definitions
+ *
+ * @author      Christian Manal <manal@uni-bremen.de>
+ * @author      Leandro Lanzieri <leandro.lanzieri@haw-hamburg.de>
+ */
+
+
+#ifndef NET_LWM2M_H
+#define NET_LWM2M_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup lwm2m_conf     LWM2M compile configurations
+ * @ingroup config
+ * @{
+ */
+
+/**
+ * @brief CoAP default port of the LWM2M server
```

Comment:
I think any text that references Lightweight M2M as a protocol in prose/comments (not code) should use the standard spelling, "LwM2M", with lower case "w". There are some cases in this file that seem to reference a collection of macros based on how the macros are defined, so "LWM2M" is OK there. Use your judgement, and don't sweat those cases too much.

Please make a sweep through the package and example code for this issue. Thanks.